.travis.yml

```diff
@@ -45,6 +46,7 @@ install:
   - gem install bundler
   - bundle install
   - git clone https://github.com/DataDog/dd-agent.git ~/dd-agent
+  - cp ~/dd-agent/datadog.conf.example ~/dd-agent/datadog.conf
```

Comment:
Small note: The reason we don't encounter this, though, is that most of us have the agent installed on our systems. So, people could encounter this outside of travis.